lib/events/auditwriter.go

```diff
@@ -457,7 +457,8 @@ func (a *AuditWriter) closeStream(stream Stream) {
 func (a *AuditWriter) completeStream(stream Stream) {
 	ctx, cancel := context.WithTimeout(a.cfg.Context, defaults.NetworkBackoffDuration)
 	defer cancel()
-	if err := stream.Complete(ctx); err != nil {
+	err := stream.Complete(ctx)
+	if err != nil {
```

Comment:
Nit: Prefer using the `if err := ...; err != nil { ... }` pattern where possible.  Keeps code more concise and is a common enough pattern to be immediately visually recognizable.